File: app/src/main/java/com/skydoves/multicolorpickerdemo/CustomFlag.java
package com.skydoves.multicolorpickerdemo;

import android.content.Context;
import android.view.View;
import android.widget.TextView;

import com.skydoves.multicolorpicker.ColorEnvelope;
import com.skydoves.multicolorpicker.FlagView;

/**
 * Developed by skydoves on 2017-10-24.
 * Copyright (c) 2017 skydoves rights reserved.
 */

public class CustomFlag extends FlagView {

    private TextView textView;
    private View view;

    public CustomFlag(Context context, int layout) {
        super(context, layout);
        textView = findViewById(R.id.flag_color_code);
        view = findViewById(R.id.flag_color_layout);
    }

    @Override
    public void onRefresh(ColorEnvelope envelope) {
        textView.setText("#" + envelope.getHtmlCode());
        view.setBackgroundColor(envelope.getColor());
    }
}

File: app/src/main/java/com/skydoves/multicolorpickerdemo/CustomFlag.kt
package com.skydoves.multicolorpickerdemo

import android.content.Context
import android.view.View
import android.widget.TextView

import com.skydoves.multicolorpicker.ColorEnvelope
import com.skydoves.multicolorpicker.FlagView

/**
 * Developed by skydoves on 2017-10-24.
 * Copyright (c) 2017 skydoves rights reserved.
 */

class CustomFlag(context: Context, layout: Int) : FlagView(context, layout) {

    private val textView: TextView
    private val view: View

    init {
        this.textView = findViewById(R.id.flag_color_code)
        this.view = findViewById(R.id.flag_color_layout)
    }

    override fun onRefresh(envelope: ColorEnvelope) {
        this.textView.text = "#" + envelope.htmlCode
        this.view.setBackgroundColor(envelope.color)
    }
}
